platform/net/arp/arping_test.go

```diff
@@ -19,7 +19,7 @@ type failingInterfaceAddress struct{}
 
 func (ia failingInterfaceAddress) GetInterfaceName() string { return "eth0" }
 
-func (ia failingInterfaceAddress) GetIP() (string, error) {
+func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error) {
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... `is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)